18F/identity-idp
# LG-14481: DocAuth infinity attempts remaining bug

## 🎫 Ticket
[LG-14481](https://cm-jira.usa.gov/browse/LG-14481)

## 🛠 Summary of changes
Do not display remaining attempts when the remainingSubmitAttempts property has not been changes and is still Infinity

## 📜 Testing Plan
1. Upload documents that would trigger a 500 response by the ImageUplioadsController#create
2. Verify that `For security reasons, you have Infinity attempts remaining to add your ID online.` is not rendered.

## 👀 Screenshots

If relevant, include a screenshot or screen capture of the changes.

<details>
<summary>Before:</summary>
<img width="731" alt="Screenshot 2024-09-13 at 5 27 42 PM" src="https://github.com/user-attachments/assets/659685f8-1d2f-4b8c-848a-849c0a752cd5">

</details>

<details>
<summary>After:</summary>
<img width="641" alt="Screenshot 2024-09-13 at 5 26 47 PM" src="https://github.com/user-attachments/assets/8f8bcd56-d79e-4ecc-baa1-dce2f829dfda">

</details>
